sections/earn/ClaimTab/ClaimTab.tsx

```diff
@@ -227,13 +227,15 @@ const ClaimTab: React.FC<ClaimTabProps> = ({ tradingRewards, stakingRewards, tot
 						gasLimit = normalizeGasLimit(gasLimit);
 					}
 
+					const normalizedGasPriceValue = Math.round(normalizedGasPrice(gasPrice.toNumber()));
```

Comment:
Hmmm this is weird. I don't think this is where it should be fixed. We might have to go where the gas price is being fetched and check what's happening

Reply:
Hmm yea I agree.. Should've dug a bit deeper. We use the result of calling: `await FeePool.estimateGas.claimFees()` as gas estimate, I guess this must have returned a number with decimals.. Dont have any account I can claim with right now :(